research/frontend/js/index.js

```diff
@@ -0,0 +1,55 @@
+$(document).ready(function () {
+    $(".modal").modal();
+  
+    $("#edit-interests").click(function (e) {
+      e.preventDefault();
```

Comment:
Tab size should be fixed to exact one size. This code uses both 2 and 4 spaces. I recommend fixing your editor config `tabSize` to **4** on Javascript.

Reply:
okay, i'll try to fix and apply it.